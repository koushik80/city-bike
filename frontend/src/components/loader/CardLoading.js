import { Card, Grid, Skeleton } from '@mui/material'
import React from 'react'

const CardLoading = () => {
  return (
    <Grid item xs={8} md={6} lg={3}>
      <Card style={{ padding: '1rem' }}>
        <Skeleton variant="text" sx={{ fontSize: '2rem' }} />
        <Skeleton variant="text" sx={{ fontSize: '2rem' }} />
        <Skeleton variant="text" sx={{ fontSize: '2rem' }} />
      </Card>
    </Grid>
  )
}

export default CardLoading