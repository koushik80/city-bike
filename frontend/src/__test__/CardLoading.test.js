import React from 'react';
import renderer from 'react-test-renderer';
import CardLoading from '../components/loader/CardLoading';


describe("Card Loading", () => {

    it("should render correctly", () => {
        const wrapper = renderer.create(<CardLoading />);

        expect(wrapper).toMatchSnapshot();
    })
});