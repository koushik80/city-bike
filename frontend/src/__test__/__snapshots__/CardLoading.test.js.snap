// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Loading should render correctly 1`] = `
<div
  className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-8 MuiGrid-grid-md-6 MuiGrid-grid-lg-3 css-13i4rnv-MuiGrid-root"
>
  <div
    className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-bhp9pd-MuiPaper-root-MuiCard-root"
    style={
      Object {
        "padding": "1rem",
      }
    }
  >
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-19vxxfm-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-19vxxfm-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
    <span
      className="MuiSkeleton-root MuiSkeleton-text MuiSkeleton-pulse css-19vxxfm-MuiSkeleton-root"
      style={
        Object {
          "height": undefined,
          "width": undefined,
        }
      }
    />
  </div>
</div>
`;
